.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.6
```

Comment:
i would also add a `.ruby-gemset` with `mossleyheritage` so it doesn't use the global gemset for ruby 2.7.6